app/test_split_integer.py

```diff
@@ -2,20 +2,24 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    goal = split_integer(16, 4)
+    assert sum(goal) == 16
```

Comment:
We can refactor it in single line. Fix in all cases.

```suggestion
    assert sum(split_integer(16, 4)) == 16
```